cybozu/tcp.cpp

```diff
@@ -27,7 +27,7 @@
 
 namespace {
 
-const unsigned int MAX_BUFCNT = 100;
+const unsigned int MAX_BUFCNT = 65536;
```

Comment:
This implies that a socket struct can have up to 64 GiB (65536 MiB) cache memory.